pkgs/os-specific/linux/tomb/default.nix

```diff
@@ -31,12 +31,13 @@ stdenv.mkDerivation rec {
     install -Dm644 doc/tomb.1 $out/share/man/man1/tomb.1
 
     wrapProgram $out/bin/tomb \
-      --prefix PATH : $out/bin:${lib.makeBinPath [ cryptsetup gettext gnupg pinentry util-linux e2fsprogs ]}
+      --prefix PATH : $out/bin:${lib.makeBinPath [ cryptsetup gettext gnupg libargon2 pinentry sudo util-linux e2fsprogs ]}
```

Comment:
How does this sudo work? I think sudo generally has to be a setuid binary provided by the system (`/run/wrappers/bin/sudo` on NixOS).

Reply:
@bjornfor Hello, thanks for the review.

Yes indeed, I modified the package to use `/run/wrappers/bin/sudo` for sudo (which is also overridable through `sudoPath`).

I also had to patch the package because tomb calls sudo through $PATH and does not support using a specific binary path for sudo command.

What do you think?